oxide-vpc/src/api.rs

```diff
@@ -276,13 +276,22 @@ pub struct PhysNet {
     pub vni: Vni,
 }
 
-/// The physical address for a guest.
+/// The physical address for a guest, minus the VNI.
+///
+/// We save space in the VPC mappings by grouping guest
+/// Virtual-to-Physical mappings by VNI.
```

Comment:
These are really helpful, I never quite understood why we split these up this way.